app/model/editions/client/ClientArticleMetadata.scala

```diff
@@ -28,7 +28,9 @@ case class ClientArticleMetadata (
   imageCutoutSrc: Option[String],
   imageCutoutSrcHeight: Option[String],
   imageCutoutSrcWidth: Option[String],
-  imageCutoutSrcOrigin: Option[String]
+  imageCutoutSrcOrigin: Option[String],
+
+  overrideArticleMainMedia: Option[Boolean]
```

Comment:
So I'd be expecting this as a `Boolean`?

Reply:
The classes that are exported to you are the ones prefixed with `Published`.